src/test/integration/run-command.ts

```diff
@@ -71,6 +71,9 @@ async function pipesToString(
   let str = '';
   const promises = [];
   for (const stream of [stdout, stderr]) {
+    if (!stream) {
```

Comment:
Huh, when would this be the case?

Reply:
Don't know why, but if you set `silent=true`, it will be undefined and throw.

Reply:
I guess the typings for node commands aren't super great about nulls, so this sort of defensiveness isn't unreasonable.